collections/individual/index.php

```diff
@@ -1086,7 +1080,14 @@ function initializeMap(){
 							if($collMetadata['contact']){
 								echo $LANG['ADDITIONAL_INFO'].': '.$collMetadata['contact'];
 								if($collMetadata['email']){
-									$emailSubject = $DEFAULT_TITLE.' occurrence: '.$occArr['catalognumber'].' ('.$occArr['othercatalognumbers'].')';
+									$otherCatNum = '';
+									if($occArr['othercatalognumbers']){
+										foreach($occArr['othercatalognumbers'] as $identArr){
+											$otherCatNum .= $identArr['value'] . ', ';
+										}
+										$otherCatNum = ' (' . trim($otherCatNum, ', ') . ')';
```

Comment:
I'm confused why we're trimming `', '`. Didn't we just add that in the above line?